redwood-layout-shared-test/src/main/kotlin/app/cash/redwood/layout/AbstractFlexContainerTest.kt

```diff
@@ -280,36 +277,34 @@ private fun ConstraintEnum.toConstraint() = when (this) {
   ConstraintEnum.Fill -> Constraint.Fill
 }
 
-enum class AlignItemsEnum {
-  FlexStart,
-  FlexEnd,
+enum class CrossAxisAlignmentEnum {
+  Start,
   Center,
-  Baseline,
+  End,
   Stretch,
 }
 
-private fun AlignItemsEnum.toAlignItems() = when (this) {
-  AlignItemsEnum.FlexStart -> AlignItems.FlexStart
-  AlignItemsEnum.FlexEnd -> AlignItems.FlexEnd
-  AlignItemsEnum.Center -> AlignItems.Center
-  AlignItemsEnum.Baseline -> AlignItems.Baseline
-  AlignItemsEnum.Stretch -> AlignItems.Stretch
+private fun CrossAxisAlignmentEnum.toCrossAxisAlignment() = when (this) {
+  CrossAxisAlignmentEnum.Start -> CrossAxisAlignment.Start
+  CrossAxisAlignmentEnum.End -> CrossAxisAlignment.End
+  CrossAxisAlignmentEnum.Center -> CrossAxisAlignment.Center
+  CrossAxisAlignmentEnum.Stretch -> CrossAxisAlignment.Stretch
 }
 
-enum class JustifyContentEnum {
-  FlexStart,
-  FlexEnd,
+enum class MainAxisAlignmentEnum {
+  Start,
   Center,
+  End,
   SpaceBetween,
   SpaceAround,
   SpaceEvenly,
 }
 
-private fun JustifyContentEnum.toJustifyContent() = when (this) {
-  JustifyContentEnum.FlexStart -> JustifyContent.FlexStart
-  JustifyContentEnum.FlexEnd -> JustifyContent.FlexEnd
-  JustifyContentEnum.Center -> JustifyContent.Center
-  JustifyContentEnum.SpaceBetween -> JustifyContent.SpaceBetween
-  JustifyContentEnum.SpaceAround -> JustifyContent.SpaceAround
-  JustifyContentEnum.SpaceEvenly -> JustifyContent.SpaceEvenly
+private fun MainAxisAlignmentEnum.toMainAxisAlignment() = when (this) {
+  MainAxisAlignmentEnum.Start -> MainAxisAlignment.Start
+  MainAxisAlignmentEnum.Center -> MainAxisAlignment.Center
+  MainAxisAlignmentEnum.End -> MainAxisAlignment.End
+  MainAxisAlignmentEnum.SpaceBetween -> MainAxisAlignment.SpaceBetween
+  MainAxisAlignmentEnum.SpaceAround -> MainAxisAlignment.SpaceAround
+  MainAxisAlignmentEnum.SpaceEvenly -> MainAxisAlignment.SpaceEvenly
 }
```

Comment:
Probably best to put the value inside the enum, like
```kotlin
enum class MainAxisAlignmentEnum(val value: MainAxisAlignment) {
  Start(MainAxisAlignment.Start)
  ...
}
```
to eliminate the need to do a mapping function.

Reply:
https://github.com/cashapp/redwood/pull/1377